.env-template

```diff
@@ -1,16 +0,0 @@
-# ENV
-
-## GraphQL Port
-PORT=4000
-
-## Google Service key
-GOOGLE_APPLICATION_CREDENTIALS=/home/user/Downloads/service-account-file.json
-
-## frontend url
-FRONTEND_URL=http://localhost:8080/
-
-## backend url
-BACKEND_URL=http://localhost:4000/
-
-## Google Cloud Storage bucket
-BUCKET="gs://nzpmc-backend.appspot.com"
```

Comment:
deleted?

Reply:
Hmm interesting... I accepted this as a current change when rebasing but since it'e been on master since May...